sparkles/tests/test_checks.py

```diff
@@ -308,6 +308,33 @@ def test_guide_count_or():
     assert msg['category'] == 'caution'
     assert 'OR with more than 1 stars brighter than 5.5.' in msg['text']
 
+    # Set a scenario with guide_count in the 3.5 to 4.0 range and confirm a
+    # critical warning.
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(n_stars=5, mag=[7.0, 7.0, 7.0, 10.2, 10.3])
+    aca = get_aca_catalog(**mod_std_info(n_fid=3, n_guide=5, obsid=1),
+                          stars=stars, dark=DARK40,
+                          raise_exc=True)
+    aca = ACAReviewTable(aca)
```

Comment:
Better to stick with the convention of `acar` for a generic review table `ACAReviewTable` and `aca` for the `ACATable`.  It just makes the code easier to read.